roles/postgres/README.md

```diff
@@ -1,12 +1,12 @@
 Postgres
 ========
 
-A role to setup Pulp 3's postgres service.
+A role to setup Pulp 3's postgres:
```

Comment:
A role to setup postgres in Pulp 3?

Reply:
I want to somehow list the kubernetes objects this role can create/manage